inc/class-wp-cli-command.php

```diff
@@ -99,6 +99,39 @@ public function save( $args, $args_assoc ) {
 			if ( ! empty( $args_assoc['verbose'] ) ) {
 				WP_CLI::line( 'Saving ' . $url );
 			}
+
+			$options = [
+				'user_id' => get_current_user_id(),
+				'context' => '',
+				'action'  => '',
+			];
+
+			$post_id = url_to_postid( $url );
+			if ( ! empty( $post_id ) ) {
+				$post = get_post( $post_id );
```

Comment:
There needs to be a check here to ensure that `$post` contains on object of the type `WP_Post`.

Reply:
Added here a97f145692a03d14060122e9cfd306cd031a160d